src/librbd/cache/pwl/Request.h

```diff
@@ -347,9 +347,9 @@ class GuardedRequest {
   }
   friend std::ostream &operator<<(std::ostream &os,
                                   const GuardedRequest &r) {
-    os << "guard_ctx->state=[" << r.guard_ctx->state << "], "
-       << "block_extent.block_start=" << r.block_extent.block_start << ", "
-       << "block_extent.block_start=" << r.block_extent.block_end;
+    os << " guard_ctx->state=[" << r.guard_ctx->state
```

Comment:
Same here.  It is output as `"req=" << req` and `submitting queued request to blockguard: " << req`, why the extra space?